source/intro.tex

```diff
@@ -2,7 +2,7 @@
 \rSec0[intro]{General}
 
 \indextext{diagnostic message|see{message, diagnostic}}%
-\indexdefn{conditionally-supported behavior!see{behavior, conditionally-supported}}%
+\indexdefn{conditionally-supported behavior|see{behavior, conditionally-supported}}%
```

Comment:
When to use indexdefn vs. indextext?  Looking in macros.tex, they seem to be equivalent.